package.json

```diff
@@ -1,9 +1,10 @@
 {
   "name": "atd-product",
-  "version": "1.0.0",
+  "version": "1.0.1",
   "private": true,
   "dependencies": {
-    "@netlify/plugin-nextjs": "^5.11.2",
```

Comment:
We don't need this plug: it is optional to include it for apps that do use the nextjs functions AND in cases where you want to pin its version.

This is also controlled by the deployment **Runtime** in Netlify. In order to fully remove this I had to go to Project configuration > Build & deploy > Build Settings, and edited the build settings by clearing the Runtime input (which was set to NextJS). This cut the build time in half :)